### PR TITLE
python dependency download instructions

### PR DESCRIPTION
**Pull request by hf-mitre**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -106,6 +109,46 @@ We use the basic feature branch GIT flow. Create a feature branch off of master
...
+
+```bash
+pip install -r caldera/requirements.txt --no-index --find-links caldera/python_deps
+```
+
+CALDERA can then be started as usual:
+
+```bash
+cd caldera
+python server.py -E ...
```
Comment (privateducky) [inline README.md]: put the actual command in full:
```
python server.py
```

File: README.md [hunk 2]
```diff
@@ -106,6 +109,46 @@ We use the basic feature branch GIT flow. Create a feature branch off of master
...
+the offline target machine runs Python 3.6 on CentOS 7, then Python3.6 and CentOS 7 should be used to perform 
+the packaging to minimize problems.
+
+```bash
+git clone --recursive https://github.com/mitre/caldera.git
+mkdir caldera/python_deps
+pip download -r caldera/requirements.txt --dest caldera/python_deps
+```
+
+If you want to use the adversary plugin, be sure to download those dependencies as well:
```
Comment (privateducky) [inline README.md]: move this section to the adversary plugin readme, under an 'offline' section

File: README.md [hunk 3]
```diff
@@ -106,6 +109,46 @@ We use the basic feature branch GIT flow. Create a feature branch off of master
 request. Make branch names and commits descriptive. A merge request should solve one problem,
 not many. 
 
+## Offline installation
```
Comment (privateducky) [inline README.md]: move this section to the bottom of the README (just above the versions section)

File: README.md (hunk 1 above)
Comment (hf-mitre) [reply README.md]: 7a217054d770574c248071b9bb52033f3a8999ab

File: README.md (hunk 2 above)
Comment (hf-mitre) [reply README.md]: 54732611b43cdeecf556daa1c4c6421dce5c7d73

File: README.md (hunk 3 above)
Comment (hf-mitre) [reply README.md]: 54732611b43cdeecf556daa1c4c6421dce5c7d73

**Pull request merged**